### PR TITLE
Release 2022 06 23

### PR DESCRIPTION
**Pull request by bitdivine**
# Motivation
We would like to publish a new version of the package.

# Changes
* Bump package version.
* Update changelog

# Tests

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 0.5.0 (2022-06-23)
+
+- ???
```
Comment (lmuntaner) [inline CHANGELOG.md:3]: My change last week:

* Change "fromSubAccountId: string" param to "fromSubAccount: number[]".

**Review by lmuntaner — CHANGES_REQUESTED** [review]
Add change:

* Change "fromSubAccountId: string" param to "fromSubAccount: number[]" in "LedgerCanister#transfer" and "GovernanceCanister#stakeNeuron".

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
+# 0.5.0 (2022-06-23)
+
+### Features
```
Comment (peterpeterparker) [inline CHANGELOG.md:3]: ### Breaking

- change "fromSubAccountId: string" param to "fromSubAccount: number[]"

### Features

- Remove `payload`, `payloadBytes` and `nnsFunctionName` fields from proposal of type ExecuteNnsFunction action (moved to nns-dapp)
- use Candid interface by default for ledger canister and proto for hardware wallet
- new function `transactionFee` to query the current transaction fee

Comment (bitdivine) [reply CHANGELOG.md]: So is this all good now?

**Review by lmuntaner — APPROVED** [review]

**Pull request merged**